eventually/mentor/views.py

```diff
@@ -30,8 +34,9 @@ def get(self, request):
 
         :return: return JsonResponse within filtered students data
         """
+
         mentor = CustomUser.get_by_id(request.user.id)
-        topics_id = [record.id for record in mentor.topic_set.all()]
+        topics_id = [record.id for record in MentorStudent.objects.all()]
```

Comment:
don't use objects in view

Reply:
have added 'get_all' method to mentor models and use it